src/data/finset/lattice.lean

```diff
@@ -357,6 +358,28 @@ begin
   refine ⟨max'_mem _ _, λ t Ht, le_antisymm (le_max' s t Ht) (le_trans a (min'_le s t Ht))⟩,
 end
 
+lemma max'_eq_of_dual_min' {s : finset α} (hs : s.nonempty) :
+  max' s hs = of_dual (min' (image to_dual s) (nonempty.image hs to_dual)) :=
+begin
+  rw [of_dual, to_dual, equiv.coe_fn_mk, equiv.coe_fn_symm_mk, id.def],
```

Comment:
Ideally you wouldn't have to unfold these to prove this, but I suppose it's way easier than the alternative